options.mk

```diff
@@ -185,7 +185,7 @@ endif
 
 
 ifeq ($(DEBUG),1)
-  CFLAGS+=-O0 -g -ggdb3 -D"DEBUG=1"
```

Comment:
Can the `DEBUG=1` remain for other projects?

Reply:
we removed the DEBUG because it will easily cause namespace conflicts. (Indeed it's re-defined by gnu-efi code and I spent half of an hour to debug that). TPM was the only user before this PR btw.